vnstatd.te

```diff
@@ -60,27 +65,35 @@ miscfiles_read_localization(vnstatd_t)
 # Client local policy
 #
 
+# dac_override : write /var/lib/vnstat/*
+allow vnstat_t self:capability dac_override;
 allow vnstat_t self:process signal;
 allow vnstat_t self:fifo_file rw_fifo_file_perms;
 allow vnstat_t self:unix_stream_socket { accept listen };
 
 manage_dirs_pattern(vnstat_t, vnstatd_var_lib_t, vnstatd_var_lib_t)
 manage_files_pattern(vnstat_t, vnstatd_var_lib_t, vnstatd_var_lib_t)
-files_var_lib_filetrans(vnstat_t, vnstatd_var_lib_t, { dir file })
 
 kernel_read_network_state(vnstat_t)
 kernel_read_system_state(vnstat_t)
 
+# read /sys/class/net/eth0
+dev_read_sysfs(vnstat_t)
+
 domain_use_interactive_fds(vnstat_t)
 
+files_dontaudit_search_home(vnstat_t)
 files_read_etc_files(vnstat_t)
+files_search_var_lib(vnstat_t)
 
 fs_getattr_xattr_fs(vnstat_t)
 
-logging_send_syslog_msg(vnstat_t)
```

Comment:
vnstat_t is the interactive user application, from my experience it does not write to syslog